lib/tjbot.js

```diff
@@ -756,20 +730,20 @@ TJBot.prototype.see = function() {
 
     // capture 'this' context
     var self = this;
-    
+
     return new Promise(function(resolve, reject) {
         temp.open('tjbot', function(err, info) {
             if (err) {
                 reject("error: could not open temporary file for writing at path: " + info.path);
             }
-            
+
             self._captureImage(info.path).then(function(buffer) {
-                console.info("> sending image to Watson Visual Recognition");
+                winston.debug("sending image to Watson Visual Recognition");
                 var params = {
                     images_file: fs.createReadStream(info.path + ".jpg"),
                     threshold: self.configuration.see.confidenceThreshold.object
                 };
-                
+
                 self._visualRecognition.classify(params, function(err, response) {
```

Comment:
yeah, i'd probably argue that returning responses that are more in line with what watson returns is better. out of scope for this PR though.  :)